package.json

```diff
@@ -20,8 +20,7 @@
     "url": "git+https://github.com/wix/wix-ui-backoffice.git"
   },
   "scripts": {
-    "build": "yoshi build && import-path --path src/components --dts",
-    "pretest": "build-storybook",
+    "build": "yoshi build && import-path --path src/components --dts && build-storybook",
```

Comment:
This means that when someone clone the repo and does `npm test` then it won't work. The storybook needs to be built.
I thought that it is a standad that any package `npm test` should be able to pass without needing to run `npm build`...

Reply:
we need to run a task after `build` but before `test` (`pr-postbuild`), so it's a tradeoff.